src/web/containers/Layout/Layout.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Head from 'next/head';
+import Layout from './Layout';
+import Container from '../../components/Container/Container';
+
+describe('Render Layout', () => {
```

Comment:
When you test a component, the main describe string should be the component name

```javascript
describe('Container', () => /* ... */)
```